### PR TITLE
MIR-857-Further_information_for_author_search

### PR DESCRIPTION
**Pull request by rochpaul**
[Link to jira](https://mycore.atlassian.net/browse/MIR-857).

### CODE REVIEW

File: mir-module/src/main/resources/META-INF/resources/js/jquery.search-entity.js [hunk 1]
```diff
@@ -421,9 +573,10 @@
       $typeMenu.attr("role", "menu");
 
       for ( var type in SearchEntity.TYPES) {
-        if (SearchEntity.TYPES[type][options.searchEntityType].enabled == false)
-          continue;
-
+      	if (SearchEntity.TYPES[type][options.searchEntityType].enabled === false || type.includes("_FALLBACK")) {
```
Comment (yagee-de) [inline mir-module/src/main/resources/META-INF/resources/js/jquery.search-entity.js:576]: Codacy found an issue: [Mixed spaces and tabs.](https://app.codacy.com/manual/MyCoRe/mir/pullRequest?prid=6286752)

**Comment by yagee-de** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 5
- Added 2
           

Clones added
============
- mir-module/src/main/resources/META-INF/resources/js/jquery.search-entity.js  16
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/MyCoRe/mir/pullRequest?prid=6286752&bid=20455321)

File: mir-module/src/main/resources/META-INF/resources/js/jquery.search-entity.js [hunk 2]
```diff
@@ -421,9 +573,10 @@
...
+      		continue;
```
Comment (yagee-de) [inline mir-module/src/main/resources/META-INF/resources/js/jquery.search-entity.js:577]: Codacy found an issue: [Mixed spaces and tabs.](https://app.codacy.com/manual/MyCoRe/mir/pullRequest?prid=6286752)

**Review by kkrebs — APPROVED** [review]
works!

**Comment by yagee-de** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 5
- Added 2
           

Clones added
============
- mir-module/src/main/resources/META-INF/resources/js/jquery.search-entity.js  16
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/MyCoRe/mir/pullRequest?prid=6286752&bid=20455321)

File: mir-module/src/main/resources/META-INF/resources/js/jquery.search-entity.js [hunk 3]
```diff
@@ -475,9 +628,15 @@
         return;
 
       var type = null;
+      var type_fallback = null;
+      
       for (var t in SearchEntity.TYPES) {
         if (this.selectedType.toUpperCase() == t.toUpperCase()) {
           type = SearchEntity.TYPES[t][options.searchEntityType];
+          if (SearchEntity.TYPES.hasOwnProperty(t + "_FALLBACK")) {
+            console.log('SearchEntity.prototype.search: Set fallback for type ' + t);
```
Comment (yagee-de) [inline mir-module/src/main/resources/META-INF/resources/js/jquery.search-entity.js:637]: Codacy found an issue: [Unexpected console statement.](https://app.codacy.com/manual/MyCoRe/mir/pullRequest?prid=6286752)

File: mir-module/src/main/resources/META-INF/resources/js/jquery.search-entity.js [hunk 4]
```diff
@@ -475,9 +628,15 @@
         return;
 
       var type = null;
+      var type_fallback = null;
```
Comment (yagee-de) [inline mir-module/src/main/resources/META-INF/resources/js/jquery.search-entity.js:631]: Codacy found an issue: [Identifier 'type_fallback' is not in camel case.](https://app.codacy.com/manual/MyCoRe/mir/pullRequest?prid=6286752)

**Comment by yagee-de** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 5
- Added 2
           

Clones added
============
- mir-module/src/main/resources/META-INF/resources/js/jquery.search-entity.js  16
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/MyCoRe/mir/pullRequest?prid=6286752&bid=20455321)

File: mir-module/src/main/resources/META-INF/resources/js/jquery.search-entity.js [hunk 5]
```diff
@@ -490,19 +648,31 @@
...
+              that.showResult(SearchEntity.sortData(input, typeof type.dataConvert == "function" ? type.dataConvert(data) : data));
+            } else {
+              that.showResult();
+            }
+            content.detach();
+            that.$searchBtn.text(text);
+          
+          }, () => {
+            if (!isFallback && typeFallback != null) {
+              console.log('SearchEntity.prototype.search: Failed to loadData for type: ' + this.selectedType.toUpperCase() + '. Set '
```
Comment (yagee-de) [inline mir-module/src/main/resources/META-INF/resources/js/jquery.search-entity.js:663]: Codacy found an issue: [Unexpected console statement.](https://app.codacy.com/manual/MyCoRe/mir/pullRequest?prid=6286752)

File: mir-module/src/main/resources/META-INF/resources/js/jquery.search-entity.js [hunk 6]
```diff
@@ -490,19 +648,31 @@
...
+                + this.selectedType.toUpperCase() + '_FALLBACK' + ' as type.');
+              type = typeFallback;
+              handleData(type.url, type.dataType, type.data(input), true);
+            } else {
+              console.error('SearchEntity.prototype.search: LoadData failed for type ' + this.selectedType.toUpperCase());
```
Comment (yagee-de) [inline mir-module/src/main/resources/META-INF/resources/js/jquery.search-entity.js:668]: Codacy found an issue: [Unexpected console statement.](https://app.codacy.com/manual/MyCoRe/mir/pullRequest?prid=6286752)

**Pull request merged**